docker/jupyterlab/Dockerfile

```
@@ -73,7 +73,7 @@ RUN echo "**** install sphinx ****" && \
     echo "**** install Dapla tools" && \
     python3 -m pip install \
     dapla-toolbelt==1.8.1 \
-    ssb-datadoc[gcs]==0.2.5 \
+    ssb-datadoc[gcs]==0.3.0 \
```

`[gcs]` extras does not exist any more, does this not produce an error?

```suggestion
    ssb-datadoc==0.3.0 \
```

---

Nice catch! It only gives a warning apparently:
`WARNING: ssb-datadoc 0.3.0 does not provide the extra 'gcs'`